api/python/provisioner/commands/deploy_jbod.py

```diff
@@ -140,6 +144,11 @@ def run(self, **kwargs):
                 self._run_states('system', run_args)
                 self._encrypt_pillar()
 
+            if 'utils' in run_args.states:
+               logger.info("Deploying foundation states")
+               self._run_states('utils', run_args)
+               self._encrypt_pillar()
```

Comment:
@83bhp    why we don't encrypt in deploy.py and deploy_vm.py?

Reply:
good point, but I am really not sure. Will explore soon.